framework/packages/abstract-core/src/native/version_control.rs

```diff
@@ -23,13 +23,12 @@ pub mod state {
     use cw_storage_plus::{Item, Map};
 
     use crate::objects::{
-        account::AccountId, common_namespace::ADMIN_NAMESPACE, module::ModuleInfo,
-        module_reference::ModuleReference, namespace::Namespace,
+        account::AccountId, module::ModuleInfo, module_reference::ModuleReference,
+        namespace::Namespace,
     };
 
     use super::{AccountBase, Config, ModuleConfiguration, ModuleDefaultConfiguration};
 
-    pub const ADMIN: Admin = Admin::new(ADMIN_NAMESPACE);
     pub const FACTORY: Admin = Admin::new("fac");
```

Comment:
Noticed that we had this unused `ADMIN` state item. And not sure why FACTORY is `Admin`, instead of `Item<Addr>`

Reply:
Because we do an assertion for the address in the contract and Admin has nice helpers for it. 

But I guess that could be confusing? Open to including it in Config as well.

Reply:
Even though it has nice helpers, imo it will be clearer to have simple `Item<Addr>` and compare to that. Also we still want to have custom error from that, instead of "not admin" on the contract that has `cw-ownable` owner